core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2909,6 +2911,37 @@ private File getRootDirFor(String name) {
         return new File(new File(getRootDir(),"jobs"), name);
     }
 
+    public static String urlEncodeMatrixConfigString(String fullName) {
+        String html_escaped_string = "";
```

Comment:
Could you please name variables using Java common guidelines, such: https://dzone.com/articles/best-practices-variable-and Take a look to other classes to get inspired. Thanks!

Reply:
Thanks for the feedback. I can work on your comments.

However, I just want to make sure that you confirm this is a valid bug, this patch is reasonable fix, and if I work on it (fix variable names, tets cases, etc...) you are willing to accept this patch and merge to master?

In between, can you please provide me the link to failing test cases. I was trying to locate it yesterday but couldn't make sense of it.

Thanks, again.

Regards,
Sendil